tests/functional/disaster-recovery/regional-dr/test_ols_recipe_generation.py

```
@@ -0,0 +1,720 @@
+        """
+        # Turn 1 — basic recipe
+        prompt_1 = _build_prompt(
+            "Generate an ODF Disaster Recovery Recipe for an application named "
+            "finance-app deployed in the finance namespace. The application "
+            "consists of one Deployment labeled app: finance-app and one PVC "
+            "labeled app: finance-app. Generate the complete Recipe."
+        )
+        result_1 = ols_client.query(prompt_1)
+        conversation_id = result_1.get("conversation_id")
+        assert conversation_id, "OLS must return a conversation_id for multi-turn tests"
+        logger.info(f"Turn 1 conversation_id: {conversation_id}")
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)